static/components/Map.js

```diff
@@ -142,6 +142,19 @@ const Map = (props) => {
     }
 
     const currentLocationMarker = currentGPSLocation ? (
+      <>
+      {/* Translucent background circle */}
```

Comment:
instead of putting comments here, you can define each marker above line 144 as a separate variable then put the two variables in this JSX block